lib/base/variables.css

```diff
@@ -48,6 +48,8 @@
   --fg3: #665c54;
```

Comment:
here you can use a more understandable name.

Reply:
sure! will do
Thanks